app/controllers/application_controller/indexes.rb

```diff
@@ -57,6 +57,15 @@ def build_index_with_query
     filtered_index(new_query, display_opts)
   end
 
+  def check_for_spider_block
+    trace_tests
+    return unless !User.current && params[:page].to_i > 10
+
+    Rails.logger.warn(:runtime_spiders_begone.t)
+    render(json: :runtime_spiders_begone.t,
+           status: :forbidden)
```

Comment:
I was tempted to use 418 (I’m a teapot) rather than 403 (Forbidden), but I resisted the urge…

Reply:
I'm in favor of 418, feel free :)